packs/st2/actions/workflows/send_anonymous_install_data.yaml

```diff
@@ -0,0 +1,37 @@
+---
+  chain:
+    -
+      name: "check_for_permission_to_send_anonymous_data"
+      ref: "st2.kv.get"
+      params:
+        key: "st2::collect_anonymous_data"
+      on-success: "get_server_uuid"
+    -
+      name: "get_server_uuid"
+      ref: "st2.kv.get"
+      params:
+        key: "st2::server_uuid"
+      publish:
+        server_uuid: "{{get_server_uuid.result}}"
+      on-success: "generate_anonymous_data"
+    -
+      name: "generate_anonymous_data"
+      ref: "core.local"
+      params:
+        cmd: "st2-submit-debug-info --exclude-configs --exclude-logs --exclude-content --review"
```

Comment:
Yeah, that actually would be a lot easier. :+1:

But, we can change it after-the-fact too if you have cycles. I didn't want to derail to add a simple flag that I could otherwise get to.